readme.md

```diff
@@ -62,6 +62,7 @@ Once you search you should see a list similar to the one below (versions may be
 * [RangeSelector](http://docs.uwpcommunitytoolkit.com/en/master/controls/RangeSelector/)
 * [RotatorTile](http://docs.uwpcommunitytoolkit.com/en/master/controls/RotatorTile/)
 * [SlideableListItem](http://docs.uwpcommunitytoolkit.com/en/master/controls/SlidableListItem/)
+* [TextBoxMask](http://docs.uwpcommunitytoolkit.com/en/master/controls/TextBoxMask/)
```

Comment:
It's not really a control, I wonder if creating an attached properties section may be more appropriate? This is more a discussion point than anything else right now.